JavaScript/JavaScriptSDK/Contracts/Generated/ContextTagKeys.ts

```diff
@@ -35,20 +38,38 @@ module AI
         public userAgent: string;
         public userId: string;
         public userStoreRegion: string;
+        public userAuthUserId: string;
+        public userAnonymousUserAcquisitionDate: string;
+        public userAuthenticatedUserAcquisitionDate: string;
         public sampleRate: string;
         public internalSdkVersion: string;
         public internalAgentVersion: string;
+        public internalDataCollectorReceivedTime: string;
```

Comment:
I think if there IS a feedback it's on schema itself not on your implementation, Assaf. Shouldn't be a blocker for this pull request but please follow up with sergkanz@microsoft.com to address his concerns.